Java/core/src/main/java/com/amazon/randomcutforest/sampler/CompactSampler.java

```diff
@@ -104,6 +107,11 @@
      */
     private final boolean storeSequenceIndexesEnabled;
 
+    /**
+     * controls the behavior of the initial set of points
```

Comment:
We need better documentation for this parameter. Maybe you could just copy the description from the constructor.

Reply:
done